luigi/execution_summary.py

```diff
@@ -276,6 +278,7 @@ def _get_comments(group_tasks):
 _COMMENTS = set((
     ("already_done", 'present dependencies were encountered'),
     ("completed", 'ran successfully'),
+    ("ever_failed", 'ever failed'),
```

Comment:
Yeah, it's just notification and not so important.
I will remove this from summary output.